understanding/20/use-of-color.html

```diff
@@ -44,6 +44,43 @@ <h2>Intent of Use of Color</h2>
          </p>
          
       </div>
+
+
+      <div class="note">
+
+         <p>If content is conveyed through the use of colors that differ not only in their hue,
+            but that also have a significant difference in lightness, then this counts as an additional
+            visual distinction, as long as the difference in contrast between the colors is 3:1 or greater.
```

Comment:
The explanation uses hue, lightness and contrast without explaining that (we define) contrast as a difference in lightness. Suggest:  "as long as the difference in lightness (contrast) between the colors is 3:1 or greater."

Reply:
to be pedantic, should it perhaps more accurately be "the difference in relative luminance between the colors leads to a contrast ratio of 3:1 or greater" (thinking of the actual glossary definition here https://www.w3.org/TR/WCAG/#dfn-contrast-ratio)

Reply:
made changes here to use the more accurate glossary definition. would be nice to link to/include the glossary definition of "contrast ratio", but beyond adding a hard link (which would then potentially link to wcag 2.0 or 2.1 or whatever, making it immediately broken when this SC is reused in other versions) i don't think there's some magic incantation to add in the definition directly/automatically in the understanding itself